cpp/src/c_api/uniform_neighbor_sampling.cpp

```diff
@@ -226,6 +229,22 @@ struct uniform_neighbor_sampling_functor : public cugraph::c_api::abstract_funct
                                                             vertex_partition_lasts,
                                                             do_expensive_check_);
 
+      std::optional<rmm::device_uvector<vertex_t>> renumber_map{std::nullopt};
+      std::optional<rmm::device_uvector<size_t>> renumber_map_offsets{std::nullopt};
+
+#if 0
```

Comment:
I assume this is disabled pending Seunghwa's PR getting merged?

Reply:
Yes.  Just pushed an update with a version of Seunghwa's branch merged in (not the latest).